src/metrics/deployments/__tests__/deployments.test.ts

```diff
@@ -40,47 +38,171 @@ describe("Deployments", () => {
     jest.useFakeTimers({ now: FAKE_NOW });
   });
 
+  beforeEach(() => {
+    Mocktokit.reset({
+      // endpoint to save json data
+      ["PUT /repos/{owner}/{repo}/contents/{path}"]: async () => undefined,
+    });
+  });
+
+  afterEach(() => {
+    expect(Mocktokit.unexpectedRequestsMade).toStrictEqual([]);
+  });
+
   it("event gets signed as deployment event", async () => {
-    octokitResponse = {
-      data: mockedDeploymentList,
-    };
     const eventBody = {
       ...mockedDeploymentEvent,
     };
 
+    Mocktokit.mocks[
+      "GET /repos/{owner}/{repo}/deployments?environment={environment}"
+    ] = async () => ({
+      headers: {},
+      data: mockedDeploymentList,
+    });
+
+    Mocktokit.mocks["GET /repos/{owner}/{repo}/contents/{path}"] =
+      async () => ({
+        headers: {},
+        data: mockedPackageJson,
+      });
+
     const result = await handler(eventBody);
 
     expect(result).toMatchObject([
       {
+        dataEventSignature: DataEventSignature.Deployment,
         created_at: FAKE_NOW,
+        status: "success",
         output: {},
-        dataEventSignature: DataEventSignature.Deployment,
       },
     ]);
   });
 
   it("sets timeSinceLastDeploy to null if there was no previous deploy on env", async () => {
-    octokitResponse = {};
     const eventBody = {
       ...mockedDeploymentEvent,
     };
 
+    Mocktokit.mocks[
+      "GET /repos/{owner}/{repo}/deployments?environment={environment}"
+    ] = async () => ({
+      headers: {},
+      data: [],
+    });
+
+    Mocktokit.mocks["GET /repos/{owner}/{repo}/contents/{path}"] =
+      async () => ({
+        headers: {},
+        data: mockedPackageJson,
+      });
+
     const result = await handler(eventBody);
 
     expect(result).toHaveLength(1);
     expect(result[0]).toMatchObject({
+      status: "success",
       output: {
-        timeSinceLastDeploy: null,
+        deployTime: 1658193553000,
+        duration: 86400000,
+        env: "production",
+        environmentDeployments: {
+          isInitialDeployment: true,
+          timeSinceLastDeploy: null,
+        },
+        packageJson: {
+          isParseable: false,
+          version: null,
```

Comment:
with the Mocktokit mock, this should be true or has a value different to null, as `mockedPackageJson` is valid json data that includes a version.